debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p /mit
+ExecStartPre=/bin/chown root:pyhesiodfs ${pyhesiodfs_dir}
+ExecStartPre=/bin/chmod 770 ${pyhesiodfs_dir}
+ExecStart=/usr/bin/pyhesiodfs -f ${pyhesiodfs_dir} -o nonempty
+ExecStopPost=/bin/umount -l ${pyhesiodfs_dir}
+
+[Install]
+WantedBy=multi-user.target remote-fs.target openafs-client.service
```

Comment:
I see. `openafs-client.service` do have `multi-user.target remote-fs.target` as its `WantedBy`, though. I originally only had `multi-user.target` (consistent with the sysvinit runlevels), and that did work fine.